<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>293</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Sky Dolly</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="topMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="moduleVisibilityWidget" native="true"/>
    </item>
    <item>
     <widget class="QWidget" name="moduleSelectorWidget" native="true">
      <layout class="QHBoxLayout" name="moduleSelectorLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="replayGroupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Record &amp;&amp; Replay</string>
      </property>
      <layout class="QGridLayout" name="replayLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QSlider" name="positionSlider">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
             <property name="pageStep">
              <number>10</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
             <property name="sliderPosition">
              <number>0</number>
             </property>
             <property name="tracking">
              <bool>true</bool>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksAbove</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Time</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTimeEdit" name="timestampTimeEdit">
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="controlButtonLayout">
           <item>
            <widget class="ActionButton" name="recordButton">
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ActionButton" name="skipToBeginButton">
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ActionButton" name="backwardButton">
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>20</height>
              </size>
             </property>
             <property name="autoRepeat">
              <bool>true</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ActionButton" name="stopButton">
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ActionButton" name="pauseButton">
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ActionButton" name="playButton">
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>18</width>
               <height>20</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ActionButton" name="forwardButton">
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>20</height>
              </size>
             </property>
             <property name="autoRepeat">
              <bool>true</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ActionButton" name="skipToEndButton">
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ActionButton" name="loopReplayButton">
             <property name="text">
              <string>Loop</string>
             </property>
             <property name="icon">
              <iconset resource="../res/UserInterface.qrc">
               <normaloff>:/img/icons/play-loop-normal.png</normaloff>
               <normalon>:/img/icons/play-loop-normal-on.png</normalon>:/img/icons/play-loop-normal.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>20</height>
              </size>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="replaySpeedGroupBox">
      <property name="title">
       <string>Replay Speed</string>
      </property>
      <layout class="QHBoxLayout" name="replaySpeedLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="moduleGroupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Module</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="flightExportMenu">
     <property name="title">
      <string>Flight &amp;Export</string>
     </property>
    </widget>
    <widget class="QMenu" name="flightImportMenu">
     <property name="title">
      <string>Flight &amp;Import</string>
     </property>
    </widget>
    <widget class="QMenu" name="locationImportMenu">
     <property name="title">
      <string>Location Import</string>
     </property>
    </widget>
    <addaction name="newLogbookAction"/>
    <addaction name="openLogbookAction"/>
    <addaction name="separator"/>
    <addaction name="optimiseLogbookAction"/>
    <addaction name="separator"/>
    <addaction name="flightImportMenu"/>
    <addaction name="flightExportMenu"/>
    <addaction name="separator"/>
    <addaction name="locationImportMenu"/>
    <addaction name="separator"/>
    <addaction name="showSettingsAction"/>
    <addaction name="showLogbookSettingsAction"/>
    <addaction name="separator"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="windowMenu">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="showFlightAction"/>
    <addaction name="showSimulationVariablesAction"/>
    <addaction name="showStatisticsAction"/>
    <addaction name="separator"/>
    <addaction name="stayOnTopAction"/>
    <addaction name="showMinimalAction"/>
   </widget>
   <widget class="QMenu" name="replayMenu">
    <property name="title">
     <string>&amp;Replay</string>
    </property>
    <widget class="QMenu" name="slowMenu">
     <property name="title">
      <string>Slow</string>
     </property>
    </widget>
    <widget class="QMenu" name="fastMenu">
     <property name="title">
      <string>Fast</string>
     </property>
    </widget>
    <addaction name="recordAction"/>
    <addaction name="stopAction"/>
    <addaction name="playAction"/>
    <addaction name="pauseAction"/>
    <addaction name="separator"/>
    <addaction name="skipToBeginAction"/>
    <addaction name="backwardAction"/>
    <addaction name="forwardAction"/>
    <addaction name="skipToEndAction"/>
    <addaction name="separator"/>
    <addaction name="slowMenu"/>
    <addaction name="fastMenu"/>
    <addaction name="normalSpeedAction"/>
    <addaction name="customSpeedAction"/>
    <addaction name="separator"/>
    <addaction name="loopReplayAction"/>
   </widget>
   <widget class="QMenu" name="moduleMenu">
    <property name="title">
     <string>&amp;Module</string>
    </property>
   </widget>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="showModulesAction"/>
    <addaction name="showReplaySpeedAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="onlineManualAction"/>
    <addaction name="separator"/>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="replayMenu"/>
   <addaction name="moduleMenu"/>
   <addaction name="viewMenu"/>
   <addaction name="windowMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <action name="quitAction">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="aboutQtAction">
   <property name="text">
    <string>About &amp;Qt...</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="showSimulationVariablesAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Simulation &amp;Variables...</string>
   </property>
   <property name="shortcut">
    <string>V</string>
   </property>
  </action>
  <action name="recordAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Record</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="playAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/UserInterface.qrc">
     <normaloff>:/img/icons/play-normal.png</normaloff>
     <normalon>:/img/icons/play-normal-on.png</normalon>
     <activeoff>:/img/icons/play-active.png</activeoff>
     <activeon>:/img/icons/play-active.png</activeon>:/img/icons/play-normal.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Play</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="pauseAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/UserInterface.qrc">
     <normaloff>:/img/icons/pause-normal.png</normaloff>
     <normalon>:/img/icons/pause-normal-on.png</normalon>
     <activeoff>:/img/icons/pause-normal-active.png</activeoff>:/img/icons/pause-normal.png</iconset>
   </property>
   <property name="text">
    <string>P&amp;ause</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="showSettingsAction">
   <property name="text">
    <string>S&amp;ettings...</string>
   </property>
  </action>
  <action name="skipToBeginAction">
   <property name="icon">
    <iconset resource="../res/UserInterface.qrc">
     <normaloff>:/img/icons/begin-normal.png</normaloff>
     <activeoff>:/img/icons/begin-active.png</activeoff>:/img/icons/begin-normal.png</iconset>
   </property>
   <property name="text">
    <string>B&amp;egin</string>
   </property>
   <property name="shortcut">
    <string>PgUp</string>
   </property>
  </action>
  <action name="backwardAction">
   <property name="icon">
    <iconset resource="../res/UserInterface.qrc">
     <normaloff>:/img/icons/backward-normal.png</normaloff>
     <activeoff>:/img/icons/backward-active.png</activeoff>:/img/icons/backward-normal.png</iconset>
   </property>
   <property name="text">
    <string>Re&amp;wind</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="forwardAction">
   <property name="icon">
    <iconset resource="../res/UserInterface.qrc">
     <normaloff>:/img/icons/forward-normal.png</normaloff>
     <activeoff>:/img/icons/forward-active.png</activeoff>:/img/icons/forward-normal.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Fast Forward</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="skipToEndAction">
   <property name="icon">
    <iconset resource="../res/UserInterface.qrc">
     <normaloff>:/img/icons/end-normal.png</normaloff>
     <activeoff>:/img/icons/end-active.png</activeoff>:/img/icons/end-normal.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;nd</string>
   </property>
   <property name="shortcut">
    <string>PgDown</string>
   </property>
  </action>
  <action name="stayOnTopAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Stay on &amp;Top</string>
   </property>
  </action>
  <action name="stopAction">
   <property name="icon">
    <iconset resource="../res/UserInterface.qrc">
     <normaloff>:/img/icons/stop-normal.png</normaloff>
     <activeoff>:/img/icons/stop-active.png</activeoff>:/img/icons/stop-normal.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Stop</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="showStatisticsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Statistics...</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="showFlightAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Flight...</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="newLogbookAction">
   <property name="text">
    <string>&amp;New Logbook...</string>
   </property>
   <property name="toolTip">
    <string>New Logbook</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="openLogbookAction">
   <property name="text">
    <string>&amp;Open Logbook...</string>
   </property>
   <property name="toolTip">
    <string>Open Logbook</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="showLogbookSettingsAction">
   <property name="text">
    <string>&amp;Logbook Settings...</string>
   </property>
  </action>
  <action name="optimiseLogbookAction">
   <property name="text">
    <string>O&amp;ptimise Logbook</string>
   </property>
   <property name="toolTip">
    <string>Optimise Logbook</string>
   </property>
  </action>
  <action name="showMinimalAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Minimal</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
  <action name="showModulesAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Modules</string>
   </property>
  </action>
  <action name="showReplaySpeedAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Replay Speed</string>
   </property>
  </action>
  <action name="onlineManualAction">
   <property name="text">
    <string>Online &amp;Manual...</string>
   </property>
  </action>
  <action name="loopReplayAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/UserInterface.qrc">
     <normaloff>:/img/icons/play-loop-normal.png</normaloff>
     <normalon>:/img/icons/play-loop-normal-on.png</normalon>:/img/icons/play-loop-normal.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Loop</string>
   </property>
   <property name="toolTip">
    <string>Loop the replay.</string>
   </property>
  </action>
  <action name="normalSpeedAction">
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
  <action name="customSpeedAction">
   <property name="text">
    <string>Custom</string>
   </property>
  </action>
  <action name="actionsdfsa">
   <property name="text">
    <string>sdfsa</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ActionButton</class>
   <extends>QPushButton</extends>
   <header location="global">Widget/ActionButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/UserInterface.qrc"/>
 </resources>
 <connections/>
</ui>
